backend/project404_t8/users/models.py

```diff
@@ -46,16 +46,16 @@ def __str__(self):
     def is_admin(self):
         return self.admin
 
-    def get_url_to_author(self):
-        author_host = self.host
-        if not author_host:
-            queryset = Server.objects.all()
-            for server in queryset:
-                if server.username == "local":
-                    author_host = server.host
-                    break
+    # def get_url_to_author(self):
```

Comment:
commented this out because I was getting an error about server not being able to be pulled into code from API.models - which is odd because API.models has a Server class